providers/tentacle.rb

```diff
@@ -281,3 +280,17 @@ def verify_roles(roles)
 def verify_environment(environment)
   raise 'Environment is required in order to register a Tentacle with Octopus Deploy Server' unless environment
 end
+
+def verify_port(port,polling)
```

Comment:
Will do, this is much better. 

The reason for using fancy_boolean is so that it is consistent with the :configure resource. 

```
action :configure do
  new_resource = @new_resource
  name = new_resource.name
  instance = new_resource.instance
  checksum = new_resource.checksum
  version = new_resource.version
  upgrades_enabled = new_resource.upgrades_enabled
  home_path = new_resource.home_path
  config_path = new_resource.config_path
  app_path = new_resource.app_path
  trusted_cert = new_resource.trusted_cert
  polling = fancy_bool(new_resource.polling)
```

Should we modify :configure and :register so that it converts boolean to fancy_boolean as late as possible? 

So this: 
      .\\Tentacle.exe configure --instance="#{instance}" --home="#{home_path}" --app="#{app_path}" --port="#{port}" --noListen="#{polling}" --console

Would become

      .\\Tentacle.exe configure --instance="#{instance}" --home="#{home_path}" --app="#{app_path}" --port="#{port}" --noListen="#{fancy_boolean(polling)}" --console